packages/back-end/types/app-features.d.ts

```diff
@@ -0,0 +1,68 @@
+/**
```

Comment:
Should likely merge this with `front-end/types/app-features` -- no need to have 2 of the same.